### PR TITLE
add option to change package:// to model:// when loading urdf file

### PR DESCRIPTION
**Pull request by k-okada**
this feature is required if we want to use gzweb, but ofcourse you have to set GAZEBO_MODEL_PATH correctly

### CODE REVIEW

**Comment by hsu** [comment]
@k-okada do you have an example? tutorial? thanks.

**Comment by 130s** [comment]
Related discussion https://bitbucket.org/osrf/gzweb/issue/45/creating-local-model-of
@hsu

**Comment by hsu** [comment]
+1

File: gazebo_ros/scripts/spawn_model [hunk 1]
```diff
@@ -243,6 +248,9 @@ class SpawnModel():
           print "Error: user specified param or filename is an empty string"
           sys.exit(0)
 
+        if self.package_to_model:
+            model_xml = re.sub("<\s*mesh\s+filename\s*=\s*\"package:/?/?","<mesh filename=\"model://", model_xml)
```
Comment (scpeters) [inline gazebo_ros/scripts/spawn_model:252]: this only works for attributes with double quotes `"`. I think it should be updated to handle single quotes `'` as well

Comment (hsu) [inline gazebo_ros/scripts/spawn_model:252]: #445

**Pull request merged**